src/components/Page/components/Header/Header.tsx

```diff
@@ -123,17 +128,33 @@ class Header extends React.PureComponent<ComposedProps, State> {
         </div>
       ) : null;
 
-    const titleMarkup = (
-      <div className={styles.Title}>
-        <div className={styles.DisplayTextWrapper}>
-          <DisplayText size="large" element="h1">
-            {title}
-          </DisplayText>
+    const titleMetadataMarkup = titleMetadata ? (
+      <div className={styles.TitleMetadata}>{titleMetadata}</div>
+    ) : null;
+
+    const subtitleMarkup = subtitle ? (
+      <Heading element="h2">{subtitle}</Heading>
```

Comment:
Maybe a header is not the best for this. I couldn't find the w3c official page on it, but I remembered this: http://html5doctor.com/howto-subheadings. Sounds like because hgroup is obsolete they recommend a p tag.

Reply:
Yea looks like `hgroup` was removed from HTML5. Interesting https://www.w3.org/TR/2014/CR-html5-20140204/common-idioms.html#sub-head

Reply:
💯